engine/src/main/java/org/terasology/rendering/dag/nodes/SimpleBlendMaterialsNode.java

```diff
@@ -44,7 +45,7 @@
 
     @Override
     public void initialise() {
-
+        addDesiredStateChange(new SetBlend(true));
```

Comment:
In line with previous comment, this should be: ` addDesiredStateChange(new EnableBlending());`